pkg/util/nsenter/nsenter.go

```diff
@@ -0,0 +1,139 @@
+// +build linux
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nsenter
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"k8s.io/utils/exec"
+
+	"github.com/golang/glog"
+)
+
+const (
+	hostRootFsPath = "/rootfs"
+	// HostProcMountsPath is the default mount path for rootfs
+	HostProcMountsPath = "/rootfs/proc/1/mounts"
```

Comment:
If `Nsenter` struct should abstract all `/bin/nsenter` calls then all these constants should be private and not exported out of the module. I still see they're used in `NsenterMounter` and that's wrong.